### PR TITLE
#1444: Add class param docs for format option

### LINKED ISSUE #1444: Custom format classes not being applied correctly
I have the following custom formats:
```
        formats: {
          h1: { block: 'h1', classes: 'title is-1' },
          h2: { block: 'h2', classes: 'title is-2' },
          h3: { block: 'h3', classes: 'title is-3' },
          h4: { block: 'h4', classes: 'title is-4' },
          h5: { block: 'h5', classes: 'title is-5' }
        }
```
When choose H1, the classes become "title is-1" - this is correct.
Then I choose H2, the classes become "title is-1 is-2" - not correct
Then I choose H3, the classes become "title is-1 is-2 is-3" - not correct
Then I choose H1 again, the classes become "title is-1 is-2 is-3"

Only the first instance is the element rendered properly; all other instances the element is rendered like H3 even though choosing H1 should remove "is-2 is-3"

### PR DESCRIPTION
**Pull request by tylerkelly13**
Resolves #1444 

Description of Changes:
* Add `class` formats parameter and clarify `classes` description.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by lnewson — CHANGES_REQUESTED** [review]
As mentioned on slack, there is no `class` configuration option for formats so this information is incorrect.

The addition about adding to classes, instead of overwriting is correct but that's it from what I can see.

File: _includes/configuration/formats.md [hunk 1]
```diff
@@ -184,6 +189,24 @@ tinymce.init({
...
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  formats: {
+    // Changes the default format for the bold button to produce a span with a bold class
+    bold: {
+      inline: 'span',
+      attributes:
+      { 'class': 'bold' }
```
Comment (lnewson) [inline _includes/configuration/formats.md:204]:
```suggestion
      attributes: { class: 'bold' }
```

**Review by lnewson — APPROVED** [review]

File: _includes/configuration/formats.md [hunk 2]
```diff
@@ -132,7 +132,12 @@ For each format some additional parameters can be specified:
 
 #### `classes`
 
-Space-separated list of classes that are applied to the selected or newly created inline/block element.
+Space-separated list of classes that are:
+
+- Applied to newly created inline or block elements.
+- Appended to selected existing inline or block elements.
```
Comment (EkimChau) [inline _includes/configuration/formats.md:138]: Appended to selected existing inline or block elements.

maybe change of order?

Appended to existing selected inline or block elements.

**Review by EkimChau — APPROVED** [review]

File: _includes/configuration/formats.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/configuration/formats.md:138]: > Appended to existing selected inline or block elements.

This reads oddly to me

Comment (tylerkelly13) [reply _includes/configuration/formats.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1831/commits/739e20d7cf784a9fb7fad6acca85d3c12d482cf9

**Pull request merged**